workers-site/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "private": true,
-  "name": "worker",
+  "name": "{{ project-name }}",
```

Comment:
this will make it so no one can ever git clone and then run wrangler. IDK if it's worth it. for other templates, I just omit `name`
```suggestion
```